### PR TITLE
gh-1642 Select asset screen

### PR DESCRIPTION
**Pull request by elgatovital**
Handles #1642

Changes proposed in this pull request:
- Add screen for selecting assets
- Add name to TokenBalance
- Filter list using name and symbol from TokenBalance

simulator_screenshot_1F372300-0445-498C-B490-4A06708C2422

### CODE REVIEW

File: Multisig/Cross-layer/Analytics/TrackingEvent.swift [hunk 1]
```diff
@@ -31,6 +31,8 @@ enum TrackingEvent: String, Trackable {
 
     case assetsNoSafe                               = "screen_assets_no_safe"
     case assetsCoins                                = "screen_assets_coins"
+    case assetsTransferSelect                       =
+        "screen_assets_transfer_select"
```
Comment (sche) [inline Multisig/Cross-layer/Analytics/TrackingEvent.swift]: could you please make it one line and add it to this doc https://docs.google.com/spreadsheets/d/1mj9iQIhpM-Pak7lQMb1dmxhQn1QdJoJyhbc730dZNzA/edit#gid=1949479240 ?

File: Multisig/UI/Assets/Transfer/SelectAssetViewController.swift [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        self.init(namedClass: Self.superclass())
+        self.balances = balances
+        self.filteredBalances = balances
+    }
+    
+    func filterTokens(term: String) {
+        self.term = term
+        
+        filteredBalances = balances.filter { balance in
+            return balance.symbol.contains(term)
```
Comment (sche) [inline Multisig/UI/Assets/Transfer/SelectAssetViewController.swift]: `return balance.symbol.contains(term) || balance.name.contains(term)`

File: Multisig/UI/Assets/Transfer/SelectAssetViewController.swift [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        tableView.allowsSelection = false
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.estimatedRowHeight = 60
+        tableView.backgroundColor = tableBackgroundColor
+        
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.refreshControl = nil
+        
+        emptyView.setImage(UIImage(named: "ico-no-assets.pdf")!)
```
Comment (sche) [inline Multisig/UI/Assets/Transfer/SelectAssetViewController.swift]: no need for `.pdf` suffix

File: Multisig/UI/Assets/Transfer/SelectAssetViewController.swift [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        onSuccess()
+        Tracker.trackEvent(.assetsTransferSelect)
+    }
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        1
```
Comment (sche) [inline Multisig/UI/Assets/Transfer/SelectAssetViewController.swift]: One is a default value. You can remove it

**Comment by sche** [comment]
👍🏻  good progress

**Review by DmitryBespalov — APPROVED** [review]
great

**Pull request merged**